### PR TITLE
README

### PR DESCRIPTION
**Pull request by akoushke**
needs work. drafting... please add your feedbacks and ideas on how to make our README file better

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+* **feat**: A new feature
+* **fix**: A bug fix
+* **perf**: A code change that improves performance
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+* **test**: Adding missing tests or correcting existing tests
+
+### Scope
+
+The scope should indicate what is being changed. Generally, these should match component or adapter names. For example, `WebexComponent`, `WebexAdapter`, etc. Other than those, `tooling` tends to be the most common.
```
Comment (adamweeks) [inline CONTRIBUTING.md]: Should we update these examples with scopes that would be relevant in this repo?

Comment (akoushke) [reply CONTRIBUTING.md]: this PR is open for discussion.

File: README.md [hunk 2]
```diff
@@ -1 +1,153 @@
-### Webex SDK Component Adapter
+<h1 align='center' style='border-bottom: none;'>Webex SDK Component Adapter</h1>
+<h3 align='center'>Webex adapter to share SDK data with the components</h3>
```
Comment (lalli-flores) [inline README.md]:
```suggestion
<h3 align='center'>Webex adapter to share SDK data with <a href="..link to components" >Webex components</a></h3>
```

File: README.md [hunk 3]
```diff
@@ -1 +1,153 @@
...
+  </a>
+  <a href='#badge'>
+    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
+  </a>
+  <a href='https://github.com/webex/sdk-component-adapter/blob/master/LICENSE'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
+
+**Webex SDK Adapter** TBD.
```
Comment (lalli-flores) [inline README.md]: What is the TBD?

File: README.md [hunk 4]
```diff
@@ -1 +1,153 @@
...
+- [Support](#support)
+- [Team](#team)
+
+## Install
+
+```bash
+npm install --save @webex/components @webex/sdk-component-adapter
+```
+
+## Usage
```
Comment (lalli-flores) [inline README.md]: I don't think you should include the `Styles` and `Adapters` sections here. Instead, you should link out the the Webex Components README usage section.
This usage section should only explain how to replace the JSON adapter, or any adapter with the SDK adapter.

File: README.md (hunk 3 above)
Comment (akoushke) [reply README.md]: I added some info.

**Pull request closed without merging**